nautobot/dcim/templates/dcim/location_migrate_data_to_contact.html

```diff
@@ -0,0 +1,101 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load helpers %}
+
+{% block title %}Migrating contact data from {{ obj_type }} {{ obj }}{% endblock %}
+{% block form %}
+    <div id="assign" class="tabcontent">
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Contact Data</strong></div>
+            <div class="panel-body">
+                {% render_field form.action %}
+                {% render_field form.location %}
```

Comment:
I have to render this form field for `similar_to_location_data` filter to work properly on `query_params`. `render field form.location as_hidden` does not work either.